jerry-core/jerry-api.h

```diff
@@ -144,6 +144,7 @@ bool jerry_value_to_boolean (const jerry_value_t);
 jerry_value_t jerry_value_to_number (const jerry_value_t);
 jerry_value_t jerry_value_to_object (const jerry_value_t);
 jerry_value_t jerry_value_to_string (const jerry_value_t);
+jerry_value_t jerry_value_to_normal_value (const jerry_value_t);
```

Comment:
what about jerry_value_remove_error_flag? I don't think people will understand the "normal" phrase.